### ISSUE TITLE
Icons and hover tips should be accessible for keyboard and screen reader users

### ISSUE THREAD

**Issue by swongCO** [ustaxcourt/ef-cms#8874]
As a keyboard-only and screen reader user, so that I can access information, I need icons and hover tips to be accessible.

### Description
Icons and hover tips/pop-ups are not accessible for keyboard only and screen readers. Most hover tips are on icons but there are others (like default browser pop-ups) that are not accessible.

Suggestion from devs: componentize icons to prevent variable styling; create a shared component with custom tooltip instead of the native one (shows on hover only) so we can control when it displays

### Acceptance Criteria
* Icons and hover tips are accessible via keyboard only and screen readers
* Determine if user documentation needs to be updated (Tenille): Y / N
